pkg/sdkserver/sdkserver.go

```diff
@@ -1066,6 +1068,12 @@ func (s *SDKServer) UpdateList(ctx context.Context, in *beta.UpdateListRequest)
 	if in == nil {
 		return nil, errors.Errorf("UpdateListRequest cannot be nil")
 	}
+	if in.List == nil || in.UpdateMask == nil {
```

Comment:
I defer to you @igooch on if this is correct for the fields -- you know more about this than I do. But wanted to check, because it _reads_ like List is required?